### PR TITLE
GEOPY-89

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-89 - Implement MT inversion in geoapps**

### CODE REVIEW

File: geoapps/drivers/base_inversion.py [hunk 1]
```diff
@@ -341,34 +342,35 @@ def get_tile_misfits(self, tiles):
             [],
         )
         for tile_id, local_index in enumerate(tiles):
```
Comment (benk-mira) [inline geoapps/drivers/base_inversion.py:344]: Whoops, get_tile_misfits should have been replaced by MisfitFactory.  This should have been deleted

**Review by benk-mira — CHANGES_REQUESTED** [review]
A quick scan

File: geoapps/drivers/base_inversion.py (hunk 1 above)
Comment (domfournier) [reply geoapps/drivers/base_inversion.py]: True, and deleted

**Review by benk-mira — APPROVED** [review]
all good

**Pull request merged**